tc-messaging/src/main/java/com/tc/object/EntityID.java

```diff
@@ -84,9 +84,6 @@ public static EntityID readFrom(TCByteBufferInput serialInput) throws IOExceptio
 
   @Override
   public String toString() {
-    return "EntityID{" +
-           "className='" + className + '\'' +
-           ", entityName='" + entityName + '\'' +
-           '}';
+    return className + ":" + entityName;
   }
```

Comment:
not sure I understand this, could you tell me what exactly needs to be changed here?